pages/jdl.md

```diff
@@ -317,7 +499,42 @@ microservice C with myotherjhipsterapp
 search * with elasticsearch except C
 ```
 
-The first option is used to tell JHipster that you want your microservice to deal with your entities, whereas the second specifies how and if you want your entities searched.
+The first option is used to tell JHipster that you want your microservice to deal with your entities, whereas the second
+specifies how and if you want your entities searched.
+
+
+### <a name="annotations"></a> Annotations
+
+Annotations are available since JHipster v5. Similarly to what's possible in Java, annotations work the same way so that
+you annotate your entities with annotations options.
+
+Take this JDL code for instance:
+```
+entity A
+entity B
+entity C
+
+dto C with mapstruct
```

Comment:
Same as above, this JDL is invalid because `service` is required for the `dto` option.  Fix is to add `service C with serviceClass`

Reply:
Fixed